### PR TITLE
Governance: code of conduct, codeowners, pull template, maintainers

### PR DESCRIPTION
**Pull request by penguinolog**
* Add typehints for protected methods

### CODE REVIEW

File: threaded/_base_gthreadpooled.py [hunk 1]
```diff
@@ -18,6 +18,7 @@
 
 import typing  # noqa  # pylint: disable=unused-import
 
+import gevent.event  # noqa  # pylint: disable=unused-import
```
Comment (kobe25) [inline threaded/_base_gthreadpooled.py]: why?

Comment (penguinolog) [reply threaded/_base_gthreadpooled.py]: It's required for valid typecheck, but not used for the main functionality.

Comment (penguinolog) [reply threaded/_base_gthreadpooled.py]: Used on L94

Comment (kobe25) [reply threaded/_base_gthreadpooled.py]: Ah yep, maybe next time better a dedicate commit for typing annotation?

**Pull request merged**